### PR TITLE
Add Gitshock Chain

### PR DESCRIPTION
**Pull request by avenbreaks**
Adding new Gitshockchain Evm

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
tl;dr: it is now required to put the binary icon into iconDownload

why: it was a PITA to always `ipfs get` in the review process - sometimes it takes *really long* or even fails (especially with pinata) - this took significant time (which adds up with the amount of chains added) and slowed down the review process. Also sometimes CIDs become unavailable after - big mess. Hence it is now required to put them in iconDownload (still named via CID)

**Comment by avenbreaks** [comment]
> tl;dr: it is now required to put the binary icon into iconDownload
> 
> why: it was a PITA to always `ipfs get` in the review process - sometimes it takes _really long_ or even fails (especially with pinata) - this took significant time (which adds up with the amount of chains added) and slowed down the review process. Also sometimes CIDs become unavailable after - big mess. Hence it is now required to put them in iconDownload (still named via CID)

sorry I forgot to add the icon to the folder, now I have added it

File: _data/icons/gitshockchain.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+      "url": "ipfs://QmW562GDDHFBsrHqbLixzpFCg4bD7NZfv9jFDfxgVfLJDY",
+      "width": 512,
+      "height": 512,
+      "format": "png"
+    }
+  ]
+
```
Comment (ligi) [inline _data/icons/gitshockchain.json:9]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]
chainId must match the filename

**Comment by avenbreaks** [comment]
> chainId must match the filename



> chainId must match the filename

other chains have used our chainid, what is the solution?

**Comment by ligi** [comment]
https://github.com/ethereum-lists/chains#collision-management

**Comment by avenbreaks** [comment]
> N

sorry we open new PR, this PR old and chainId already exist.

**Pull request closed without merging**